sam-mcp-server/src/agents/mcp_server/mcp_server_agent_component.py

```diff
@@ -194,7 +195,7 @@ def _initialize_server(self):
                     server_params = StdioServerParameters(
                         command=parts[0],
                         args=parts[1:],
-                        env=None,  # Use default safe environment
+                        env=os.environ,  # Use current process environment
```

Comment:
Basically copying the environment into every MCP server. This is not the best from a security perspective, but having a fine grained environment for each MCP server will require a bit of a re-write.